floor.go

```diff
@@ -105,19 +109,39 @@ func (f *Floor) watchFloorPrice() {
 				continue
 			}
 
+			// calculate if price has moved up or down
+			var priceRaw float64
+			if priceRaw, err = strconv.ParseFloat(strings.ReplaceAll(price, " SOL", ""), 64); err != nil {
```

Comment:
Not every floor price is in SOL. Instead of trying to go from a string to a float here you should change the getFloorPrice function to return the raw price (in float) and do the comparison on that. maybe have it also return the currency as a string so we can use that.

Reply:
I agree. I did it this way because I didn't want to change your code too much, but you are right about the currency change. 👍🏻

Reply:
@rssnyder I think we can now merge this PR. Let me know if it requires more changes.